easybuild/easyconfigs/c/Chemaxon_Marvin/Chemaxon_Marvin-21.14.eb

```diff
@@ -0,0 +1,69 @@
+##
+# This is a contribution from SIB Swiss Institute of Bioinformatics
+# Homepage:     https://www.sib.swiss/research-infrastructure/competence-centers/vital-it
+#
+# Authors::     Sebastien Moretti <sebastien.moretti@sib.swiss>
+#
+##
+
+# rpm file extension does not look to be known yet: Unknown file type for file *.rpm (at easybuild/tools/filetools.py)
+# easyblock = 'Rpm'
```

Comment:
Hmm, you should be able to use the `Rpm` easyblock for this.

Are you still seeing this unpack problem with recent versions of EasyBuild?

Reply:
In fact rpmrebuild was missing on my side.
But it still does not work: 
```
error: Summary field must be present in package: (main package)
error: License field must be present in package: (main package)
```